CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+- Updated splk-py-trace and splk-py-trace-bootstrap commands.
+  - Both commands now delegate to opentelemetry-instrument and opentelemetry-bootstrap commands.
+  - Removed support for `--exporters` CLI flag from `splk-py-trace-bootstrap` command.
+  - Added support for `--access-token` and `--service-name` to `splk-py-trace` command.
+  - `start_tracing()` was moved from `splunk_otel.tracing` to `splunk_otel`.
+  [#34](https://github.com/signalfx/splunk-otel-python/pull/34)
```

Comment:
IMO it would be more readable to keep the list flat and mention the same PR.
Especially that some points should be in `### Added` and others in `### Changed` section.

Reply:
Updated.